src/app/calculator/steam/stack-loss/stack-loss-form/stack-loss-by-volume/stack-loss-by-volume.component.ts

```diff
@@ -50,6 +50,7 @@ export class StackLossByVolumeComponent implements OnInit {
         }
       }
     }
+    this.setFuelTemp();
```

Comment:
You can delete this call and method definition. CalcExcessAir() is called on init